### DISCUSSION TITLE
Parent Grain Reconstruction

### DISCUSSION THREAD

**Discussion by ly801** [mtex-toolbox/mtex#1327] [Ask Anything]
Hi all,

I have tried for so many times, but encounter some issues for 'Parent Beta Phase Reconstruction in Titanium Alloys'. When I run the command 'job.calcVariantGraph', the MATLAB just says '' Unrecognized method, property, or field 'calcVariantGraph' for class 'parentGrainReconstructor' ''. One of my friends can run smoothly with the old version coding, however, she also encounters errors with the command 'job.calcVariantGraph'. Could anyone help me to solve this problem please? Thank you so much!

Regards,
Lu.

**Comment by nyyssont** [comment]
Hello,

it seems to work for me. Are you sure you are running MTEX 5.8.0?

If you are, can you go to the folder:
\mtex-5.8.0\EBSDAnalysis\@parentGrainReconstructor

and see if you can find the file "calcVariantGraph.m"?

**Reply by ly801** [reply]
Thank you so much for reply. I will have a try for MTEX 5.8.0.
